src/open_inwoner/components/templates/components/File/File.html

```diff
@@ -1,13 +1,22 @@
-{% load i18n link_tags utils icon_tags form_tags dropdown_tags button_tags string_tags %}
+{% load i18n l10n link_tags utils icon_tags form_tags dropdown_tags button_tags string_tags %}
 
 <aside class="file" aria-label="{% trans "Bestand" %} {{ name }}">
     <div class="file__file">
-    {% if is_image %}
-        {% icon icon="image" outlined=True %}
-    {% else %}
-        {% icon icon="description" outlined=True %}
-    {% endif %}
+        <p class="file__symbol">
+            {% if is_image %}
+                {% icon icon="image" outlined=True %}
+            {% else %}
+                {% icon icon="description" outlined=True %}
+            {% endif %}
+        </p>
+
+
     <p class="p">
```

Comment:
Remove all `<BR>'s` and set this in CSS as:
`display: flex;      flex-direction: column;`